lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1517,7 +1517,7 @@ private function validateAndCompleteTypedFieldMapping(array $mapping): array
                 ! isset($mapping['type'])
                 && ($type instanceof ReflectionNamedType)
             ) {
-                if (PHP_VERSION_ID >= 80100 && ! $type->isBuiltin() && enum_exists($type->getName(), false)) {
+                if (PHP_VERSION_ID >= 80100 && ! $type->isBuiltin() && enum_exists($type->getName())) {
```

Comment:
Not sure why autoload was disabled in the first place. @beberlei , do you remember?

Reply:
Triggering the autoloader here is correct. We cannot be certain at this point that the property type has been autoloaded already.

However, this has nothig to do with the namespace we place the enum in.

Reply:
It does, I've checked this before writing it: with disabled autoloading Enum from the Entity namespace is somehow visible, that's why unit tests working with `Enums\TypedCard` are green. But moving Enum to another namespace break things.

Reply:
Can confirm we need removing `false`. Works for me now.